addons/pos_restaurant/models/pos_config.py

```diff
@@ -13,6 +13,7 @@ class PosConfig(models.Model):
     iface_printbill = fields.Boolean(string='Bill Printing', help='Allows to print the Bill before payment.')
     iface_orderline_notes = fields.Boolean(string='Internal Notes', help='Allow custom Internal notes on Orderlines.')
     floor_ids = fields.Many2many('restaurant.floor', string='Restaurant Floors', help='The restaurant floors served by this point of sale.')
+    note_ids = fields.Many2many('pos.note', string='Note Models', help='The predefined notes of this point of sale.')
```

Comment:
why did you introduce the note_ids field in pos_restaurant when you're using it in point_of_sale module? You should probably need to move this to point_of_sale.